File: src/main/java/kr/apo2073/tnliv/utilities/Debugger.java
package kr.apo2073.tnliv.utilities;

public class Debugger {
    private boolean isDebug=false;
    public Debugger(boolean isDebug) {
        this.isDebug=isDebug;
    }
    public void debug(String message) {
        System.out.println(message);
    }

    public boolean isDebug() {
        return isDebug;
    }
}

File: src/main/kotlin/kr/apo2073/onAir/utils/Debugger.kt
package kr.apo2073.onAir.utils

import kr.apo2073.onAir.OnAir
import org.bukkit.Bukkit

class Debugger {
    companion object {
        private val plugin= OnAir.plugin
        @JvmStatic
        fun debug(string: String) {
            plugin.reloadConfig()
            if (!ConfigSet.debug) return
            OALogger().info("[DEBUG] $string")
        }
    }
}